apps/src/templates/projects/ProjectsGallery.tsx

```diff
@@ -18,7 +20,12 @@ interface ProjectsGalleryProps {
 const ProjectsGallery: React.FunctionComponent<ProjectsGalleryProps> = ({
   limitedGallery,
 }) => {
-  const [selectedTab, setSelectedTab] = useState(Galleries.PRIVATE);
+  const dispatch = useAppDispatch();
+
+  const selectedGallery = useSelector(
```

Comment:
nit: mind adding a comment that this redux store isn't typed yet, which is why we can't use `useAppSelector`?